tests/unit/common/test_config.py

```diff
@@ -0,0 +1,126 @@
+import unittest
+
+import os
+
+from ims.common import config
+
+config.load()
```

Comment:
NIT: I prefer having a main method instead of loading externally. Its upto you though.